frontend/src/metabase-lib/lib/metadata/Database.ts

```diff
@@ -137,42 +183,8 @@ export default class Database extends Base {
     });
   }
 
-  nativeQuery(native) {
```

Comment:
Unused method. Ideally, we'd probably remove all of these methods from `Database` at some point, and put them in some sort of utility function file.